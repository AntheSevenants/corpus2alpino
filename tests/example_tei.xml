<TEI 
    xmlns="http://www.tei-c.org/ns/1.0">
    <teiHeader>
        <fileDesc>
            <titleStmt>
                <title>Goedendag!</title>
                <author>Sheean Spoel</author>
            </titleStmt>
            <editionStmt>
                <edition>
                    <date>2018-02-27T10:32:08.110594794</date>
                </edition>
            </editionStmt>
            <publicationStmt>
                <p>no publication statement available</p>
            </publicationStmt>
            <sourceDesc>
                <p>Written by OpenOffice</p>
            </sourceDesc>
        </fileDesc>
        <revisionDesc>
            <listChange>
                <change>
                    <name>Sheean Spoel</name>
                    <date>2018-02-27T10:33:38.490559117</date>
                </change>
            </listChange>
        </revisionDesc>
    </teiHeader>
    <text xml:id="foo">
        <body>
            <head xml:id="bar">Goedendag!</head>
            <p>Dit is een voorbeeldtekst om te demonstreren dat het mogelijk is tekst om te zetten vanuit 
                <hi rend="italic">TEI-formaat </hi>naar 
                <hi rend="italic">Alpino/Lassy-formaat</hi>. Een 
                <hi rend="underline">fijne</hi>  dag nog!
            </p>
        </body>
    </text>
</TEI>