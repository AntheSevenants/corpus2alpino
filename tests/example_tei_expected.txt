##META text fileDesc::titleStmt::title = Goedendag!
##META text fileDesc::titleStmt::author = Sheean Spoel
##META text fileDesc::editionStmt::edition = 2018-02-27T10:32:08.110594794
##META text fileDesc::publicationStmt = no publication statement available
##META text fileDesc::sourceDesc = Written by OpenOffice

##META text id = bar
##META text tei-tag = head
bar|Goedendag ! 


##META text id = foo
##META text tei-tag = p
##META text rend = italic
foo|Dit is een voorbeeldtekst om te demonstreren dat het mogelijk is tekst om te zetten vanuit TEI-formaat naar Alpino / Lassy-formaat . 


##META text rend = underline
foo_1|Een fijne dag nog ! 


